GraphAlgorithms/LargestCliqueNaiveFinder.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphAlgorithms
+{
+    public static class LargestCliqueNaiveFinder
+    {
+        public static void Find(Graph graph, int n, List<int> clique, int v, ref List<int> largestClique)
```

Comment:
Maybe have the public method only take graph and return largestClique and call this method from the new one.
I think that having these parameters like n, clique and v is confusing since we always set them to the same values anyways

Reply:
Ok, I changed it as Karol wrote. New method Find(Graph graph) calls the old one and return largest clique.